jerry-debugger/jerry-client-ts/src/lib/cdt-proxy.ts

```diff
@@ -186,7 +186,7 @@ export class ChromeDevToolsProxyServer {
    * Sends Debugger.paused event for the current debugger location
    */
   sendPaused(breakpoint: Breakpoint | undefined, backtrace: Array<Breakpoint>,
-             reason: 'exception' | 'debugCommand' | 'other') {
+             reason: 'exception' | 'debugCommand' | 'other', exception?: string) {
```

Comment:
I know it was here already but it's started to draw attention to itself now :)
Maybe create a type for the reason close to the place where the `sendPaused` delegate contract is declared to avoid this long local type here?

`type PausedReason = 'exception' | 'debugCommand' | 'other';`

and then here:

`reason: PausedReason`

Reply:
This isn't really a delegate call but an API... the controller calls this. I'm assuming it's still a good idea but I guess I'll just declare it here.